### ISSUE TITLE
await token is stuck 

### ISSUE THREAD

**Issue by abodehq** [thiamsantos/invisible-grecaptcha#15]
dears,
i really like your component but the problem i am not able to get the token itis stuck in the await line.
i also noticed that google recaptcha not shown in the bottom right. can you please help how to fix

i am using react client.
thaaaaaaaaaaaaaaank. 
const token = await execute('my-key');
 console.log("----->"+token);

**Comment by thiamsantos** [comment]
@abodehq could you please provide some context? Which version of the module are you using? What's appearing in the console? Are you transpiling with Babel?

Maybe would be helpful if you could provide a codepen showing the error.

**Comment by thiamsantos** [comment]
@abodehq closing issue because of inactivity. Feel free to reopen if you still is having this issue.

**Comment by robertbullen** [comment]
I know what is going on here, and it's already been fixed in PR #14 and merged into the master branch, but not yet published to NPM. But for the sake of posterity, here is the explanation.

Here's the code for `execute()` in version 1.0.3:

```javascript
export function execute(sitekey, {locale = 'en', position = 'bottomright'} = {}) {
  return new Promise((resolve, reject) => {
    validateRequired(sitekey, 'sitekey')

    if (!isString(sitekey)) {
      reject(buildParamError('sitekey', 'string'))
    }

    if (!isString(position)) {
      reject(buildParamError('position', 'string'))
    }

    if (!isString(locale)) {
      reject(buildParamError('locale', 'string'))
    }

    return new Promise((resolve, reject) => {
      if (window.grecaptcha) {
        render({sitekey, position, resolve, reject})
      } else {
        loadScript(locale)
          .then(() => {
            render({sitekey, position, resolve, reject})
          })
          .catch(reject)
      }
    })
  })
}
```

Notice that it returns a promise that might invoke `reject` immediately, but will never invoke `resolve`! Instead, it creates a nested promise that invokes its own `resolve` (a shadowed variable name) upon success (indirectly within the call to `render()`). But chaining of promises doesn't work this way, and so callers of `execute()` will await on the outer promise indefinitely, meanwhile the inner promise will be silently resolved and disappear.

**Comment by robertbullen** [comment]
This is probably why the demo isn't currently working. I also observed this bug in my own project.
